transport/cluster.go

```diff
@@ -248,16 +248,15 @@ func (c *Cluster) loop() {
 		select {
 		case <-c.refreshChan:
 			c.tryRefresh()
-			ticker.Stop()
-			ticker.Reset(refreshInterval)
+			ticker = time.NewTimer(refreshInterval)
 		case <-c.reopenControlChan:
 			c.reopenControl()
 		case <-c.closeChan:
 			c.handleClose()
 			return
 		case <-ticker.C:
 			c.tryRefresh()
-			ticker.Reset(refreshInterval)
+			ticker = time.NewTimer(refreshInterval)
```

Comment:
Use [Ticker](https://pkg.go.dev/time#Ticker) not here but instead of Timer.

Ticker != Timer